docs/source/asr/configs.rst

```diff
@@ -1120,3 +1120,160 @@ Depending on the type of model, there may be extra steps that must be performed
 
 * CTC Models - `Examples directory for CTC Models <https://github.com/NVIDIA/NeMo/blob/stable/examples/asr/asr_ctc/README.md>`_
 * RNN Transducer Models - `Examples directory for Transducer Models <https://github.com/NVIDIA/NeMo/blob/stable/examples/asr/asr_transducer/README.md>`_
+
+Multi-task AED Models
```

Comment:
Is the following information present in the docs:
* What is Canary model 
* How to load a canary model and transcribe? 
* How to finetune a Canary model to adapt to new languages
* Same languages but new data -> tokenizer update
* Recommended hyper parameters for finetuning